packages/mobile/src/fees/selectors.ts

```diff
@@ -1,52 +1,25 @@
 import BigNumber from 'bignumber.js'
-import { createSelector } from 'reselect'
 import { FeeType } from 'src/fees/actions'
 import { RootState } from 'src/redux/reducers'
 import { divideByWei } from 'src/utils/formatting'
 
-const getInviteFeeEstimateInWei = (state: RootState) => state.fees.estimates.invite.feeInWei
-const getSendFeeEstimateInWei = (state: RootState) => state.fees.estimates.send.feeInWei
-const getExchangeFeeEstimateInWei = (state: RootState) => state.fees.estimates.exchange.feeInWei
-const getReclaimEscrowFeeEstimateInWei = (state: RootState) =>
-  state.fees.estimates.reclaimEscrow.feeInWei
-
-export function getFeeDollars(feeInWei: BigNumber | string) {
-  const adjustedFee = divideByWei(
-    feeInWei instanceof BigNumber ? feeInWei.toString() : feeInWei,
-    18
-  )
-  return new BigNumber(adjustedFee)
-}
-
-const feeEstimateDollarsSelectorFactory = (feeSelector: (state: RootState) => string | null) => {
```

Comment:
I understand the intention with these selectors, I believe it's not worth it to memoize these values given that it's just a single multiplication. I believe the mem costs outweight the cpu cost.

Reply:
Ahh, I see. Good catch, I'll restore.